marklogic-data-hub/build.gradle

```diff
@@ -71,13 +71,13 @@ dependencies {
     testImplementation 'org.skyscreamer:jsonassert:1.5.0'
     testImplementation 'org.hamcrest:hamcrest-junit:2.0.0.0'
     testImplementation 'org.easymock:easymock:3.4'
-    testImplementation 'ch.qos.logback:logback-classic:1.2.4'
     testImplementation 'org.slf4j:log4j-over-slf4j:1.7.31'
 
     // How MLCP handles errors in a transform changed between 10.0.4.2 and 10.0.5, such that these errors do not result
     // in records counting as "failures". This seems like a bug with MLCP. For now, 10.0.4.2 is still used for testing
     // as RunConnectedStepsWithMlcpTest expects transform errors to be counted as failures.
-    testImplementation("com.marklogic:mlcp:10.0.4.2") {
+    // We changed to the latest version with story DHFPROD-8201
+    testImplementation("com.marklogic:mlcp:10.0.8.2") {
```

Comment:
It looks like we will need to leave this one as 10.0.4.2 for now. If you can revert this change, I'll create a separate ticket to update the mlcp version here at some point.